include/cantera/thermo/Elements.h

```diff
@@ -84,6 +84,50 @@ namespace Cantera
 //! stable state at 298.15 K and 1 bar.
 #define ENTROPY298_UNKNOWN -123456789.
 
+//! Get a vector of the atomic symbols of the elements defined in Cantera.
+//! @since New in version 3.0
+const std::vector<std::string>& elementSymbols();
+
+//! Get a vector of the names of the elements defined in Cantera.
+//! @since New in version 3.0
+const std::vector<std::string>& elementNames();
+
+//! Get a map with the element symbols as keys and weights as values.
+/*!
+ * This is a constant in the application so it is only generated once
+ * when it is first needed.
+ *
+ * @since New in version 3.0
+ */
+const std::map<std::string, double>& elementSymbolToWeight();
+
+//! Get a map with the element names as keys and weights as values.
+/*!
+ * This is a constant in the application so it is only generated once
+ * when it is first needed.
+ *
+ * @since New in version 3.0
+ */
+const std::map<std::string, double>& elementNameToWeight();
```

Comment:
How would I do that, and ensure that `map::find` works properly? In Python, it would be very awkward, you'd have to do something like a tuple as the key, I think... Or are you thinking that weights would be duplicated in the map?

Reply:
> weights would be duplicated in the map?

Correct. In essence, I am just suggesting to merge the two maps - the naming becomes more intuitive, and you don't have to search two maps.

Reply:
That's an interesting idea. This would definitely simplify the locations where you have to search for the weights in multiple locations.